habitat-hitl/habitat_hitl/core/client_message_manager.py

```diff
@@ -16,8 +16,11 @@
 DEFAULT_VIEWPORT_SIZE: Final[List[float]] = [0.0, 0.0, 1.0, 1.0]
 MAIN_VIEWPORT: Final[int] = -1
 
+# TODO: Move here.
+from habitat_hitl.core.ui_elements import UICanvasUpdate
 
-# TODO: Move to another file.
+
+# TODO: Deprecated legacy UI system.
```

Comment:
Old UI elements will gradually be replaced.

These classes will be removed when all dependencies are gone.